fn_query_tor_network/fn_query_tor_network/util/customize.py

```diff
@@ -0,0 +1,98 @@
+# (c) Copyright IBM Corp. 2010, 2018. All Rights Reserved.
+# -*- coding: utf-8 -*-
+
+"""Generate the Resilient customizations required for fn_query_tor_network"""
+
+from __future__ import print_function
+from resilient_circuits.util import *
+
+def codegen_reload_data():
+    """Parameters to codegen used to generate the fn_query_tor_network package"""
+    reload_params = {"package": u"fn_query_tor_network",
+                    "incident_fields": [], 
+                    "action_fields": [], 
+                    "function_params": [u"tor_search_data"], 
+                    "datatables": [], 
+                    "message_destinations": [u"fn_tor"], 
+                    "functions": [u"fn_tor"], 
+                    "phases": [], 
+                    "automatic_tasks": [], 
+                    "scripts": [], 
+                    "workflows": [],
```

Comment:
We're almost there! The workflow and rule are missing here. But you can easily add them by doing this:

cd /path/to/resilient-community-apps
resilient-circuits codegen --reload --rule "name of rule" --workflow api_name_of_workflow

Reply:
ok will do that..